src/client/web/WebRTC.ts

```diff
@@ -0,0 +1,114 @@
+import Peer from 'peerjs'
+import Network from '../services/Network'
+
+export default class WebRTC {
+  private myPeer: Peer
+  private peers = {}
```

Comment:
Use Map<> instead of object.
This is a tricky topic in JS, both object {} and Map<> are considered "hash map", like dictionary in python, so theoretically this is the same runtime complexity O(1) lookup. However when the number of keys gets larger, JS object performance becomes really bad compared to Map.
TL;DR: just use Map when you want a hash map.

Reply:
Cool, I wanted ask you about this but forgot. Have changed it to Map.